src/pages/proposals/ProposalList.vue

```diff
@@ -449,7 +449,7 @@ q-page.page-proposals
 
   .row.q-py-md(v-if="$q.screen.gt.md")
     .col-9
-      base-placeholder.q-mr-sm(v-if="!this.proposals.length" title= "No Proposals" subtitle="Your organization has not created any proposals yet. You can create a new proposal by clicking the button below."
+      base-placeholder.q-mr-sm(v-if="!this.proposals.length && !this.filteredStagedProposals.length" title= "No Proposals" subtitle="Your organization has not created any proposals yet. You can create a new proposal by clicking the button below."
```

Comment:
@evgenibir you don't need `this.` when accessing data in component props

Reply:
Can =you remove it from `!this.proposals` and `this. filteredStagedProposals`?

Reply:
@arsenijesavic, fixed, I corrected the widget display logic, I had to take into account a lot of conditions, because there are two widgets on the page: one to create a prop, if there are none created, neither staged nor active, and the other if there are props, but do not satisfy the filtering conditions, also if one of the them, the other one shouldn't, so the "if" turned out to be slightly "ugly"

Reply:
@evgenibir got it! maybe you can upgrade it to computed var something like `hasProposals` ?

Reply:
ready